app/src/main/java/me/banes/chris/tivi/home/HomeActivity.kt

```diff
@@ -63,7 +63,11 @@ class HomeActivity : TiviActivity() {
                             supportFragmentManager.popBackStackImmediate()
                         }
                     } else {
-                        // TODO scroll to top of main fragment
+                        val fragment = supportFragmentManager.fragments[1]
```

Comment:
I'm not sure this is the correct way to fetch this. `findFragmentById(R.id.home_content)` is more appropriate.